book-3-web-api/chapters/REACT_WITH_API.md

```diff
@@ -0,0 +1,210 @@
+# Adding a React Client
+
+The cool thing about exposing a Web API is that any client capable of making HTTP requests can communicate with it. You can create web apps in React, Angular, or Vue and they could all talk to the same API. It even extends past browser based clients. You could make a mobile or desktop app and they could talk to the API. Maybe even IOT devices like microwaves or refrigerators want to make requests to your API.
+
+## Create React App
+ 
+Programming refrigerators may come in a later chapter. For now, lets build a react client that will make requests to our API. Open your Gifter project and make a directory called `wwwroot`. This is where we'll put our React code. In your terminal, `cd` into the wwwroot directory and run `npx create-react-app client`
```

Comment:
I think the `client` folder should be outside of `wwwroot`. Maybe later we can teach them to build to `wwwroot`.